examples/Makefile

```diff
@@ -104,6 +104,22 @@ build: fmt ## Build all the userspace example code.
 generate: ## Run `go generate` to build the bytecode for each of the examples.
 	go generate ./...
 
+.PHONY: build-userspace-images
```

Comment:
Probably fine if we think this will mostly be used from Github workflow, but if the user will call a lot, may want to shorten all the names from *-userspace-images to *-us-images

Reply:
I wen't ahead and did this here 👍